server/shared/stepBuilder.go

```diff
@@ -400,6 +400,7 @@ func metadataPipelineFromModelPipeline(pipeline *model.Pipeline, includeParent b
 func SanitizePath(path string) string {
 	path = filepath.Base(path)
 	path = strings.TrimSuffix(path, ".yml")
+	path = strings.TrimSuffix(path, ".yaml")
```

Comment:
can we just rm leading `.` and then rm all fine endings?

Reply:
Would have to be the other way round (first trim (yaml|yml) then the dot since it works right to left), I'm unsure about the performance implications of string-manipulation in go... If you want it with trim . then (yaml|yml) I'll implement it

Reply:
hm well lets just do it as it is now ... I might push something alter ...